test/raft/README.md

```diff
@@ -0,0 +1,17 @@
+# Running tests with pytest
```

Comment:
It would be nice if (later is fine, but not much later please) you or one of the developers of this test suite can include a section introducing this test suite. In particular, it appears that this directory contains C++ tests using the Boost test framework. So why is these tests are not just in test/boost? It could be a subdirectory test/boost/raft to keep that organization.

Reply:
I have no idea why it's on a separate directory. I don't see any issues to having it as a subdirectory in test/boost. But since I'm not developing tests, I'm leaving it as it is.

Reply:
So I am @kostja to consider doing this later. I already asked for this more than a year ago, in https://github.com/scylladb/scylladb/issues/16026

The thing is, all the work you did in this patch - adding a README.md and conftest.py - is all unnecessary if we just move it into a subdirectory of boost/

Reply:
That's true. On the other side, it took me 10–15 minutes to do so. Not a big waste of time in any case.

Reply:
There are two reasons test suites exist today: 
- different test type/configuration/suite.yaml
- different team owns the test.

We should eliminate the need for the second reason with subdirectories, and reduce the need for the first reason with conversion to pytest.